packages/web-console/src/scenes/Import/ImportCSVFiles/table-schema-dialog.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useEffect, useState } from "react"
+import { ProcessedFile } from "./types"
+import { Button } from "@questdb/react-components"
+import { Box } from "../../../components/Box"
+import { Text } from "../../../components/Text"
+import styled from "styled-components"
+import { Table as TableIcon, Edit } from "styled-icons/remix-line"
+import { Undo } from "styled-icons/boxicons-regular"
+import { Form } from "../../../components/Form"
+import { SchemaColumn } from "utils"
+import { TableSchemaColumns } from "./table-schema-columns"
+import { Drawer } from "../../../components/Drawer"
+
+const StyledTableIcon = styled(TableIcon)`
+  color: ${({ theme }) => theme.color.foreground};
+`
+
+const Content = styled(Box).attrs({ gap: "0", flexDirection: "column" })`
+  width: 100%;
+  overflow: auto;
+`
+
+const partitionByOptions = ["NONE", "HOUR", "DAY", "MONTH", "YEAR"]
+
+type FormValues = {
+  schemaColumns: SchemaColumn[]
+  partitionBy: string
+  timestamp: string
+}
+
+type Props = {
+  open: boolean
+  onOpenChange: (openedFileName: string | undefined) => void
+  onSchemaChange: (values: FormValues) => void
+  file: ProcessedFile
+}
+
+export const TableSchemaDialog = ({
+  file,
+  open,
+  onOpenChange,
+  onSchemaChange,
+}: Props) => {
+  const name = file.table_name ?? file.fileObject.name
+  const [defaults, setDefaults] = useState<FormValues>({
+    schemaColumns: [],
+    partitionBy: "NONE",
+    timestamp: "",
+  })
+
+  useEffect(() => {
+    if (file.schema) {
+      setDefaults({
+        schemaColumns: file.schema,
+        partitionBy: file.partitionBy,
+        timestamp: file.timestamp,
+      })
+    }
+  }, [file])
+
+  const columnCount = defaults.schemaColumns.length
+
+  return (
+    <Drawer
+      title={
+        <Box gap="0.5rem">
+          <StyledTableIcon size="20px" />
+          <Text color="foreground">Table schema</Text>
+        </Box>
+      }
+      open={open}
+      trigger={
+        <Button
+          skin={columnCount > 0 ? "transparent" : "secondary"}
+          prefixIcon={
+            columnCount > 0 ? <Edit size="18px" /> : <TableIcon size="18px" />
+          }
+          onClick={() => onOpenChange(name)}
+        >
+          {columnCount > 0
+            ? `${columnCount} column${columnCount > 1 ? "s" : ""}`
+            : "Add"}
+        </Button>
+      }
+    >
+      <Form<FormValues>
+        name="table-schema"
+        defaultValues={defaults}
+        onSubmit={(values) => {
+          onSchemaChange(values)
+          onOpenChange(undefined)
+        }}
+      >
+        <Box gap="0" flexDirection="column">
+          <Content>
+            <Drawer.GroupHeader>
+              <Text color="foreground">Settings</Text>
+            </Drawer.GroupHeader>
```

Comment:
```suggestion
            <Drawer.GroupHeader>
              <Text color="foreground">Settings</Text>
            </Drawer.GroupHeader>
```

i think there's no need for a header, if there's only one setting